### ISSUE TITLE
Importing model with polymorphic belongs_to raises NameError

### ISSUE THREAD

**Issue by zorab47** [zdennis/activerecord-import#475]
Importing models with polymorphic belongs_to associations attempt to load the association's class (which isn't a real class). Below it attempts to load `Discount::Discountable` which doesn't exist and raises an uninitialized constant error.

This is likely due to the changes in #470.

## Test Case

A test case exercising the issue:

```ruby
context "importing model with polymorphic belongs_to" do
  it "works without error" do
    book     = FactoryGirl.create :book
    discount = Discount.new(discountable: book)

    Discount.import([discount])

    assert_equal 1, Discount.count
  end
end
```

which yields the following error:

```
  1) Error:
#<Class:0x000055d266ebcc90>#test_#import importing model with polymorphic belongs_to_works without error:
NameError: uninitialized constant Discount::Discountable
    /home/charles/.gem/ruby/2.3.5/gems/activerecord-4.2.10/lib/active_record/inheritance.rb:158:in `compute_type'
    /home/charles/.gem/ruby/2.3.5/gems/activerecord-4.2.10/lib/active_record/reflection.rb:295:in `compute_class'
    /home/charles/.gem/ruby/2.3.5/gems/activerecord-4.2.10/lib/active_record/reflection.rb:291:in `klass'
    /home/charles/.gem/ruby/2.3.5/gems/activerecord-4.2.10/lib/active_record/reflection.rb:339:in `association_primary_key'
    /home/charles/Development/zorab47/activerecord-import/lib/activerecord-import/import.rb:717:in `block in load_association_ids'
    /home/charles/Development/zorab47/activerecord-import/lib/activerecord-import/import.rb:713:in `each'
    /home/charles/Development/zorab47/activerecord-import/lib/activerecord-import/import.rb:713:in `load_association_ids'
    /home/charles/Development/zorab47/activerecord-import/lib/activerecord-import/import.rb:455:in `block in import_helper'
    /home/charles/Development/zorab47/activerecord-import/lib/activerecord-import/import.rb:453:in `map'
    /home/charles/Development/zorab47/activerecord-import/lib/activerecord-import/import.rb:453:in `import_helper'
    /home/charles/Development/zorab47/activerecord-import/lib/activerecord-import/import.rb:394:in `import'
    /home/charles/Development/zorab47/activerecord-import/test/import_test.rb:553:in `block (3 levels) in <top (required)>'

205 runs, 424 assertions, 0 failures, 1 errors, 0 skips
rake aborted!
Command failed with status (1)
/home/charles/.gem/ruby/2.3.5/gems/rake-12.2.1/exe/rake:27:in `<top (required)>'
/home/charles/.gem/ruby/2.3.5/gems/bundler-1.16.0/lib/bundler/cli/exec.rb:75:in `load'
/home/charles/.gem/ruby/2.3.5/gems/bundler-1.16.0/lib/bundler/cli/exec.rb:75:in `kernel_load'
/home/charles/.gem/ruby/2.3.5/gems/bundler-1.16.0/lib/bundler/cli/exec.rb:28:in `run'
/home/charles/.gem/ruby/2.3.5/gems/bundler-1.16.0/lib/bundler/cli.rb:424:in `exec'
/home/charles/.gem/ruby/2.3.5/gems/bundler-1.16.0/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/home/charles/.gem/ruby/2.3.5/gems/bundler-1.16.0/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/home/charles/.gem/ruby/2.3.5/gems/bundler-1.16.0/lib/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
/home/charles/.gem/ruby/2.3.5/gems/bundler-1.16.0/lib/bundler/cli.rb:27:in `dispatch'
/home/charles/.gem/ruby/2.3.5/gems/bundler-1.16.0/lib/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
/home/charles/.gem/ruby/2.3.5/gems/bundler-1.16.0/lib/bundler/cli.rb:18:in `start'
/home/charles/.gem/ruby/2.3.5/gems/bundler-1.16.0/exe/bundle:30:in `block in <top (required)>'
/home/charles/.gem/ruby/2.3.5/gems/bundler-1.16.0/lib/bundler/friendly_errors.rb:122:in `with_friendly_errors'
/home/charles/.gem/ruby/2.3.5/gems/bundler-1.16.0/exe/bundle:22:in `<top (required)>'
/home/charles/.gem/ruby/2.3.5/bin/bundle:22:in `load'
/home/charles/.gem/ruby/2.3.5/bin/bundle:22:in `<main>'
Tasks: TOP => test:postgresql
(See full trace by running task with --trace)
```

**Comment by jkowens** [comment]
Thanks for reporting and providing a test case!

**Comment by jkowens** [comment]
This fix has been released in v0.21.0 🎉
